src/test/java/no/sikt/sws/IndexHandlerTest.java

```diff
@@ -145,4 +146,23 @@ void shouldThrowBadRequestWhenGivenIndexBeginningWithUnderscore() throws IOExcep
 
         assertThat(response.getStatusCode(), is(equalTo(HTTP_BAD_REQUEST)));
     }
+
+    @Test
+    void shouldThrowBadRequestWhenUsingNonWhitelistedCharacters() throws IOException {
+
+        var pathparams = Map.of(
+                RESOURCE_IDENTIFIER, TEST_INDEX_1 + ":illegalchar"
+        );
+
+        var request = new HandlerRequestBuilder<Void>(JsonUtils.dtoObjectMapper)
+                .withHttpMethod(HttpMethod.PUT.toString())
+                .withPathParameters(pathparams)
+                .withAuthorizerClaim(SCOPE_CLAIM,TEST_SCOPE)
+                .build();
```

Comment:
We have this code duplicated several places now, perhaps we should look into extracting it as a TestUtil method soon? (Dont need in this PR)